### ISSUE TITLE
Geez transliteration 

### ISSUE THREAD

**Issue by akadaw** [BetaMasaheft/Documentation#2433]
Dear Eugenia and pietro
I am having a problem with some Geez transcriptions like, "ä",  "qʷ"  and the capitalisation of the first letters. 
could you help me to figure it out? 
thank you
Aklilu A. Tecleab

**Comment by thea-m** [comment]
Thank you for the question, @akadaw !
The Beta masaheft guidelines for transliteration can be found here: https://betamasaheft.eu/Guidelines/?id=transliteration-principles
Could you describe the concrete problem that you're facing in more detail? Which Geez words do you want to transliterate and have problems with?
Dorothea

**Comment by akadaw** [comment]
Dear Dorothea
 thank your information.
I am having problems with capital letters at the moment as some of my
problems are answered, as I just look at the guidelines now, the
problem with, ጉዕዞ, should be guʿǝzo instead it gives,  guʿzo same with
ማእሰረ, māʾsara.  for example,  ጸሎት በእንተ ማእሰረ አጋንንት,   ṣalot baʾnta māʾsara
ʾagānnt, but for now I need ṣ to make it capital without missing the dot
below it.
thank you again.
Aklilu

**Comment by thea-m** [comment]
Thank you for the precision! Could you confirm that what you are talking about is this tool https://betamasaheft.github.io/transliteration/ ? 
This tool carries out a mechanical conversion of fidal into Latin script. This can then be used as a starting point for the correct transliteration. The tool does not carry out transliteration. The researcher always has to copy the mechanically converted text and edit it manually.
To address your examples and issues:
- The tool does not take into consideration capitalisation at all. The researcher has to edit the mechanically converted text and insert capitals wherever appropriate.
- Fidal does not differentiate between "consonant" and "consonant followed by shwa (ǝ)".  For example, in the word አጋንንት, the first ን stands for "nǝ", and the second ን stands for "n" (if you pronounce the word aloud, you can probably notice the difference). The correct transliteration is _ʾagānǝnt_. The transliteration tool https://betamasaheft.github.io/transliteration/ converts all consonants into a Latin consonant and never inserts shwa. Therefore the mechanical conversions of the tool are nearly never correct, because the researcher has to manually insert ǝ wherever it is appropriate. To know whether ǝ should be inserted after a consonant, pronounce the word aloud and listen well whether you hear it. If you are not sure, check Wolf Leslau's Comparative Dictionary of Ge'ez, which has transliteration for all entries.
- Fidal also does not differentiate between simple consonant and double (geminated) consonant. The tool always converts into simple consonants, but this has to be manually corrected by the researcher. For example, ዛቲ will be converted as "zāti", which is a wrong transliteration. The researcher has to manually correct to "zātti" with two t.

So for your example phrase, ጸሎት በእንተ ማእሰረ አጋንንት, the tool produces the automatic conversion "ṣalot baʾnta māʾsara
ʾagānnt". This can be used by you as a starting point to produce your own transliteration, but it can never be directly used as a transliteration, because manual corrections are necessary. Changing the first letter to a capital and adding ǝ in the correct places has to be done by the researcher (you) manually, and will produce: _Ṣalot baʾǝnta māʾsaraʾagānǝnt_, a correct transliteration.

To summarize, https://betamasaheft.github.io/transliteration/ can be used as a working tool, but we cannot directly copy the conversions it produces and use them as transliteration. We always have to manually correct them, checking them with Leslau's dictionnary if we are not certain.

Please let me know if this answers your questions.

**Comment by akadaw** [comment]
Thank you for the clarification. Really appreciate it.
